### PR TITLE
chore: remove dpi

### PR DESCRIPTION
**Pull request by skylarbarrera**
Fixes APP-1098

## What changed (plus any additional context for devs)
removing DPI

### CODE REVIEW

File: src/hooks/useDPI.ts [hunk 1]
```diff
@@ -1,32 +0,0 @@
-import { useQuery } from '@tanstack/react-query';
-import { isEmpty } from 'lodash';
-import { useCallback } from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { emitAssetRequest } from '../redux/explorer';
-import { AppState } from '../redux/store';
-import { IndexToken } from '@/entities';
-import { getDPIBalance } from '@/handlers/dispersion';
```
Comment (benisgold) [inline src/hooks/useDPI.ts:8]: we can remove this handler from the source file as well

File: src/components/expanded-state/unique-token/UnderlyingAsset.js [hunk 2]
```diff
@@ -1,126 +0,0 @@
-import React, { useCallback } from 'react';
-import LinearGradient from 'react-native-linear-gradient';
-import { useSelector } from 'react-redux';
-import { useRemoveNextToLast } from '../../../navigation/useRemoveNextToLast';
```
Comment (benisgold) [inline src/components/expanded-state/unique-token/UnderlyingAsset.js:4]: can also delete this

File: src/components/cards/DPICard.tsx [hunk 3]
```diff
@@ -1,123 +0,0 @@
...
-    });
-  }, [nativeCurrency, navigate, routeName]);
-
-  return (
-    <ColorModeProvider value="darkTinted">
-      <GenericCard
-        gradient={CARD_GRADIENT}
-        color="#8D65FA"
-        onPress={handlePress}
-        testID="dpi-button"
```
Comment (benisgold) [inline src/components/cards/DPICard.tsx:71]: could remove e2e test for this, it's skipped for now but could save trouble later

File: src/components/cards/DPICard.tsx [hunk 4]
```diff
@@ -1,123 +0,0 @@
...
-import { useAccountSettings } from '@/hooks';
-import store from '@/redux/store';
-import { DPI_ADDRESS } from '@/references';
-import Routes from '@/navigation/routesNames';
-import { ethereumUtils } from '@/utils';
-import { GenericCard, Gradient } from './GenericCard';
-import { ORB_SIZE } from './reusables/IconOrb';
-import { useRoute } from '@react-navigation/native';
-
-const TRANSLATIONS = i18n.l.cards.dpi;
```
Comment (benisgold) [inline src/components/cards/DPICard.tsx:26]: alsoooo these can be removed

**Review by benisgold — COMMENTED** [review]
nits, im sorry

**Review by walmat — APPROVED** [review]

File: src/components/expanded-state/unique-token/UnderlyingAsset.js (hunk 2 above)
Comment (skylarbarrera) [reply src/components/expanded-state/unique-token/UnderlyingAsset.js:4]: leaving this one could be useful

**Pull request merged**